README.md

```diff
@@ -37,12 +38,35 @@ regenerate Go code after you change the `sourcegraph.proto` file.
    git checkout proto3
    go install ./...
    ```
-1. **Install [gen-mocks](https://sourcegraph.com/sourcegraph/gen-mocks)** by running:
+
+3. **Install [gen-mocks](https://sourcegraph.com/sourcegraph/gen-mocks)** by running:
 
    ```
    go get -u sourcegraph.com/sourcegraph/gen-mocks
    ```
 
+4. **Install `gopathexec`**:
+
+   ```
+   go get -u github.com/shurcooL/gopathexec
+   ```
+
+#### OS X-specific
+
+```
+brew install gnu-sed --with-default-names
```

Comment:
Do other OS X tools/scripts rely on non-GNU sed behavior? This could mess up people's computers if so.